roles/commcare_analytics/templates/nginx/superset_nginx_without_ssl_config.j2

```diff
@@ -15,6 +15,8 @@ log_format combined_with_timing '$remote_addr - $remote_user [$time_local] '
                                 '"$request" $request_time $status $bytes_sent '
                                 '"$http_referer" "$http_user_agent"';
 
+limit_req_zone $binary_remote_addr zone=ip_limited:10m rate=50r/s;
```

Comment:
Note:

Here we are adding a "zone" with name ip_limited which will store required details from requests by a specific IP address ("binary_remote_addr" is the binary representation of an IP address, preferred to be used by Nginx due to lower memory footprint) and we are allowing a shared memory of 10MB to this storage which can be shared by different nginx worker processes.
State information for about 16,000 IP addresses takes 1 megabyte, so our zone can store about 160,000 addresses. 
We are setting a limit of 50 requests per second for this zone.

Reply:
Ya, 10MB seemed small enough, so just went with the example from the documentation only.

> is a bold ambition for the CCA userbase.

You never know :)